datasets/wmt_mlqe_task3/wmt_mlqe_task3.py

```diff
@@ -0,0 +1,252 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""WMT MLQE Shared task 3."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import glob
+import logging
+import os
+
+import datasets
+
+
+_CITATION = """
+Not available.
+"""
+
+_DESCRIPTION = """\
+This shared task (part of WMT20) will build on its previous editions
+to further examine automatic methods for estimating the quality
+of neural machine translation output at run-time, without relying
+on reference translations. As in previous years, we cover estimation
+at various levels. Important elements introduced this year include: a new
+task where sentences are annotated with Direct Assessment (DA)
+scores instead of labels based on post-editing; a new multilingual
+sentence-level dataset mainly from Wikipedia articles, where the
+source articles can be retrieved for document-wide context; the
+availability of NMT models to explore system-internal information for the task.
+
+The goal of this task 3 is to predict document-level quality scores as well as fine-grained annotations.
+"""
+
+_HOMEPAGE = "http://www.statmt.org/wmt20/quality-estimation-task.html"
+
+_LICENSE = "Unknown"
+
+_URLs = {
+    "train+dev": "https://github.com/deep-spin/deep-spin.github.io/raw/master/docs/data/wmt2020_qe/qe-task3-enfr-traindev.tar.gz",
+    "test": "https://github.com/deep-spin/deep-spin.github.io/raw/master/docs/data/wmt2020_qe/qe-enfr-blindtest.tar.gz",
+}
+
+
+_ANNOTATION_CATEGORIES = [
+    "Addition",
+    "Agreement",
+    "Ambiguous Translation",
+    "Capitalization",
+    "Character Encoding",
+    "Company Terminology",
+    "Date/Time",
+    "Diacritics",
+    "Duplication",
+    "False Friend",
+    "Grammatical Register",
+    "Hyphenation",
+    "Inconsistency",
+    "Lexical Register",
+    "Lexical Selection",
+    "Named Entity",
+    "Number",
+    "Omitted Auxiliary Verb",
+    "Omitted Conjunction",
+    "Omitted Determiner",
+    "Omitted Preposition",
+    "Omitted Pronoun",
+    "Orthography",
+    "Other POS Omitted",
+    "Over-translation",
+    "Overly Literal",
+    "POS",
+    "Punctuation",
+    "Shouldn't Have Been Translated",
+    "Shouldn't have been translated",
+    "Spelling",
+    "Tense/Mood/Aspect",
+    "Under-translation",
+    "Unidiomatic",
+    "Unintelligible",
+    "Unit Conversion",
+    "Untranslated",
+    "Whitespace",
+    "Word Order",
+    "Wrong Auxiliary Verb",
+    "Wrong Conjunction",
+    "Wrong Determiner",
+    "Wrong Language Variety",
+    "Wrong Preposition",
+    "Wrong Pronoun",
+]
+
+
+class WmtMlqeTask1(datasets.GeneratorBasedBuilder):
+    """WMT MLQE Shared task 3."""
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name=f"plain_text",
+            version=datasets.Version("1.1.0"),
+            description=f"Plain text",
+        )
+    ]
+
+    def _info(self):
+        features = datasets.Features(
+            {
```

Comment:
Here one example corresponds to one document
But it looks like each column correspond to the features for each segment.
Should we consider having one example = one segment instead ?

Let me know what you think

Reply:
I think it is crucial to keep the document structure to be able to model the long-range dependencies.